measures/BuildExistingModel/measure.rb

```diff
@@ -89,18 +84,26 @@ def run(model, runner, user_arguments)
     if not File.exists? buildstock_csv
 
       runner.registerInfo("Generating buildstock.csv sampling results.")
-    
-      require 'json'
-      # analysis = JSON.parse(File.absolute_path(File.join(File.dirname(__FILE__), "..", "..", "analysis.json")))
-      # runner.registerInfo(analysis)
-      num_datapoints = 100 # TODO: NUMDATAPOINTS=`awk -F\"maximum\": 'NF>=2 {print $2}' analysis.json | sed 's/,//g' | head -n1 | xargs` # Yes, this is gross.
+
+      num_datapoints = nil
+      File.readlines(File.absolute_path(File.join(File.dirname(__FILE__), "..", "..", "analysis.json"))).each do |line|
+        if matches = line.match(%r{\s+"maximum":\s+(?<num_datapoints>\d+)})
```

Comment:
Super gross. Why not use the ruby json library? (I had to use awk/sed/etc because I was parsing from a bash script.)

Reply:
I originally tried that, but got an "unrecognized token" or something. I'll try again.

Reply:
Weird. That might suggest it's a malformed json. If that's the case, we should at least get it fixed upstream.